diagnosis/getkuberneteslogs.sh

```diff
@@ -29,24 +49,27 @@ trap restore_ssh_config EXIT
 function download_scripts
 {
     ARTIFACTSURL=$1
-    mkdir -p $SCRIPTSFOLDER
-    
-    echo "[$(date +%Y%m%d%H%M%S)][INFO] Pulling dependencies from this repo: $ARTIFACTSURL"
-    
-    for script in common detectors collectlogs collectlogsdvm hosts
-    do
-        if [ -f $SCRIPTSFOLDER/$script.sh ]; then
-            echo "[$(date +%Y%m%d%H%M%S)][INFO] Dependency '$script.sh' already in local file system"
-        fi
-        
-        curl -fs $ARTIFACTSURL/diagnosis/$script.sh -o $SCRIPTSFOLDER/$script.sh
+    #mkdir -p $SCRIPTSFOLDER
+    if [ "$(ls -A $SCRIPTSFOLDER)" ]; then
```

Comment:
use `-d` to test if dir exists
https://www.tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html

also, the fact that the directory exists does not imply that the scripts are already in the file system